docs(ExpansionPanel): Add initial documentation (#3737)
telerik/blazor-docs: _contentTemplates/expansionpanel/notes.md

#comparison-with-panelbar

* The PanelBar is a [databound component](slug:panelbar-data-binding-overview). The ExpansionPanel and its content are [defined declaratively](slug:expansionpanel-overview#creating-blazor-expansionpanel).

> * When using a hierarchy of parent and child items, the PanelBar uses different styling for the children, while the nested ExpansionPanel instances look in the same way as their parent.

Reviewer: ```suggestion
* When using a hierarchy of parent and child items, the PanelBar uses different styling for the children, while the nested ExpansionPanel instances look like their parent.
```
